### PR TITLE
Pedant changes to support `chef-zero`

### PR DESCRIPTION
**Pull request by randomcamel**

### CODE REVIEW

**Comment by danielsdeleo** [comment]
Agree w/ @stevendanna here, it's unclear to me what was wrong before.

**Comment by jkeiser** [comment]
@stevendanna @danielsdeleo I've updated the commit message, and added in the "run pedant against chef-zero on every checkin" checkin to boot. I'll send through wilson this morning.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 .DS_Store
 erln8.config
 config.yml
+.bundle
+vendor
+Gemfile.lock
```
Comment (sdelano) [inline .gitignore]: Unless you're willing to fix everything that breaks every time something gets checked into chef/chef-zero (or any other dependent gem), please do not remove the Gemfile.lock from a piece of software that we use to run integration tests (both in CI and on-site with customers).

What benefit do we get from unlocking the gems? There's a very good reason there's a Gemfile.lock in this repo.

**Comment by jkeiser** [comment]
Wilson has passed. Merging.

**Pull request merged**